### DISCUSSION TITLE
TVBS新聞網 追蹤 快更新iOS16.2！5大功能上線 可自動過濾垃圾訊息

### DISCUSSION THREAD

**Discussion by Zoe06060606** [1101-GS4719-C/Python-Community-GS4719-C#902] [General]
這次的更新多了許多之前從未有的功能 但對於我這個se2使用者好幾個功能都不能使用 看來是時後換手機了

https://tw.news.yahoo.com/%E5%BF%AB%E6%9B%B4%E6%96%B0ios16-2-5%E5%A4%A7%E5%8A%9F%E8%83%BD%E4%B8%8A%E7%B7%9A-%E5%8F%AF%E8%87%AA%E5%8B%95%E9%81%8E%E6%BF%BE%E5%9E%83%E5%9C%BE%E8%A8%8A%E6%81%AF-043537898.html

**Comment by CPCpyt** [comment]
我覺得功能非常實用，對於垃圾訊息平常真的是不堪其擾。
身為果粉感覺真的很幸福。
